tests/tensorflow2/test_keras.py

```diff
@@ -548,16 +550,18 @@ def test_include_collections(out_dir, tf_eager_mode):
         include_collections=include_collections,
         reduction_config=ReductionConfig(norms=ALLOWED_NORMS, reductions=ALLOWED_REDUCTIONS),
     )
+    hook.get_collection("custom_optimizer_variables").include("Adam")
```

Comment:
nice, can you write one more test case, 
so that mode save config is set in collection to save tensors in eval mode rather than TRAIN mode.
hook.get_collection("custom_optimizer_variables").save_config.mode_save_config 

Expectation should be num of tensors saved should be 0.

Reply:
Added an additional assert.

```
        assert (
            len(
                trial.tensor_names(
                    collection=CollectionKeys.OPTIMIZER_VARIABLES, mode=ModeKeys.EVAL
                )
            )
            == 0,
            "No Optimizer Variables Should be Saved in EVAL Mode",
        )
```